src/index.html

```diff
@@ -4,13 +4,43 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+       initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      action="https://github.com/Dimoshua/layout_search-bar-airbnb.git"
+      method="post"
+      class="search--bar"
+    >
+      <div data-qa="big">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="big--search--bar"
+          data-qa="keypress"
+        >
+      </div>
+    </form>
+
+    <form
+      action="https://github.com/Dimoshua/layout_search-bar-airbnb.git"
+      method="post"
+      class="search--bar"
+    >
+      <div data-qa="small">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="small--search--bar"
```

Comment:
```suggestion
          class="input input--small"
```
in this case, it is not necessary to have similar names with container, you can change it to `input` for example.  Also, it would be better to have all common styles for `input` and add styles that are different to another class `input--small` - **please check CHECKLIST P2**